common/src/main/java/org/keycloak/common/Profile.java

```diff
@@ -63,6 +63,8 @@ public enum Feature {
 
         LOGIN_V1("Legacy Login Theme", Type.DEPRECATED, 1),
 
+        THEME_CONFIG("WYSIWYG theme configuration tool", Type.EXPERIMENTAL, 1),
```

Comment:
I like the name, "Easy Theme" instead of just "Theme Config".  WDYT?

```suggestion
        EASY_THEME("WYSIWYG theme configuration tool", Type.EXPERIMENTAL, 1),
```

Reply:
You really want to call it easy theme?